Demo/imdb/imdb_utils.py

```diff
@@ -0,0 +1,60 @@
+import csv
```

Comment:
What was the purpose of splitting imdb_demo.py to imdb_demo.py and imdb_utils.py?

Reply:
The reason I splited those 2 files is because I am using the populate function also in the tests.

Reply:
Same replay as with social

Reply:
same as with imdb